src/script/lua_api/l_server.cpp

```diff
@@ -45,6 +45,17 @@ int ModApiServer::l_get_server_status(lua_State *L)
 	return 1;
 }
 
+// print(text)
+int ModApiServer::l_print(lua_State *L)
```

Comment:
I'd solve both issues with lua instead, is that ok? E.g. what is now in builtin/init.lua?

```

1.